src/js/utils/id3Parser.js

```diff
@@ -24,6 +24,9 @@ define([
         i = startingIndex || 0;
         while (i < len) {
             c = array[i++];
+            if (c === 0x00 || c === 0x03) {
```

Comment:
A comment here would make it clear why we're skipping these characters:
0x00 = NULL
0x03 = EOL (end of line)